src/claimList.js

```diff
@@ -14,38 +25,223 @@ if (document.readyState === "loading") {
 }
 
 function executeOverride() {
-  const allClaims = getMetadata();
-  updateIcon();
   setupAnalytics();
-  logView(allClaims);
+  makeMobileFriendly();
+
+  const { claims, name } = getMetadata();
+  removeOldHtml();
+  addNewHtml(name, claims);
+
+  styleBody();
   addFeedbackLink();
+  updateIcon();
+  logView(claims);
 }
 
 function getMetadata() {
-  return Array.from(
-    document.getElementsByTagName("table")[2]?.children[0]?.children
-  )
+  const tableElements = document.getElementsByTagName("table");
+  const claims = Array.from(tableElements[2]?.children[0]?.children)
     .slice(1)
     .map((row, i) => ({
-      claimType: row.children[1].innerText.trim(),
-      claimDate: row.children[2].innerText.trim(),
-      claimStatus: row.children[3].innerText.trim(),
+      seqNum: row.children[0].innerText.trim(),
+      type: row.children[1].innerText.trim(),
+      date: row.children[2].innerText.trim(),
+      status: row.children[3].innerText.trim(),
     }));
+  const name =
+    tableElements[1]?.children[0]?.children[0]?.children[0]?.innerText;
```

Comment:
You can also use `rows` here as well  -> `const name = tableElements[1]?.rows[0]?.children[0]?.innerText`

Feels less necessary to break out name table here though :)